caniusepython3/overrides.json

```diff
@@ -3,10 +3,12 @@
     "bcdoc": "https://github.com/boto/bcdoc/blob/master/tox.ini",
     "certifi": "https://pypi.python.org/pypi/certifi",
     "collective.recipe.template": "https://pypi.python.org/pypi/collective.recipe.template",
+    "diamond": "https://pypi.python.org/pypi/diamond",
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
     "fabric": "https://pypi.python.org/pypi/Fabric3",
+    "filechunkio": "https://pypi.python.org/pypi/filechunkio",
```

Comment:
Please update this to have the value point to the commit that shows Python 3 support.